tools/ci_build/github/azure-pipelines/templates/react-native-ci.yml

```diff
@@ -129,7 +129,7 @@ jobs:
   - script: |
       python3 tools/python/run_android_emulator.py \
         --android-sdk-root $(ANDROID_SDK_ROOT) \
-        --create-avd --system-image "system-images;android-31;default;x86_64" \
+        --create-avd --system-image "system-images;android-30;default;x86_64" \
```

Comment:
looks like android-31 emulator does not work with the current set up yet but the combination of android-30+default works fine in the CI.